runtime/bindings/python/src/pyopenvino/CMakeLists.txt

```diff
@@ -0,0 +1,66 @@
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+project (pyopenvino)
+
+if(DEFINED OpenVINO_SOURCE_DIR)
+    set(InferenceEngine_LIBRARIES inference_engine)
+else()
+    find_package(InferenceEngine REQUIRED)
+    set(InferenceEngine_LIBRARIES IE::inference_engine)
+endif()
```

Comment:
I believe current python is based on new OpenVINOConfig.cmake. Can we use it?

Reply:
done